torch/_inductor/codegen/cuda/cuda_kernel.py

```diff
@@ -126,14 +134,59 @@ def def_kernel(
         arg_defs, *_ = self.args.cpp_argdefs()
         return f"PT_EXPORT int {self.kernel_name}({', '.join(arg_defs)}, {self._EXTRA_CPP_ARGS})"
 
-    def call_kernel(self, name: str, node: CUDATemplateBuffer) -> None:
+    @staticmethod
+    def merge_ir_inputs(
```

Comment:
Where is this function called?

Reply:
Not at all, it's going to be used in phase two. It's a converted utility function that originally was a member of CUDATemplateBuffer. I already know I will need it later, so I didn't delete it.

Reply:
This needs tests if it isn't going to be called here.

Reply:
Please remove it from this PR since it's irrelevant here and the existing PR is already super large.